components/ai_chat/resources/page/components/main/style.module.scss

```diff
@@ -15,6 +15,69 @@
   overflow: hidden;
 }
 
+.dragDetectionOverlay {
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
```

Comment:
nit:
```suggestion
  inset: 0;
```

Reply:
Fixed in 895ad8b2440d0525057938b30fb2e936ce5e3190